includes/User.php

```diff
@@ -506,7 +506,13 @@ public static function newFromToken( \WebRequest $request ) {
 					} else {
 						WikiaLogger::instance()->error( 'Helios user id mismatch', [
 							'sessionUserId' => $sessionUserId,
-							'userId' => $tokenInfo->user_id
+							'heliosResponse' => [
+								'userId' => $tokenInfo->user_id,
+								'accessToken' => $tokenInfo->access_token,
```

Comment:
I would only log part of the access token (3-5 of first and last characters)

Reply:
isn't it the same token that we send from MW to Helios in get parametes to so it is present in access logs and MediaWiki http request logs?

Reply:
Yes it is but it is not good practice and I would not like to add more places like this. We would need to clean it up eventually but adding more to the already existing ones won't help us.

Reply:
We agreed offline to keep it this way (which is the same as in other places) to make logs search easier during the bug investigation. This log will be removed once we find and fix the culprit.